.github/workflows/ci-cd.yml

```diff
@@ -119,7 +120,8 @@ jobs:
 
     steps:
     - name: Decide whether the needed jobs succeeded or failed
-      uses: re-actors/alls-green@release/v1
+      # yamllint disable-line rule:line-length
+      uses: re-actors/alls-green@05ac9388f0aebcb5727afa17fcccfecd6f8ec5fe  # release/v1
```

Comment:
Dependabot won't work with branches, only Git tags.

Reply:
Good to know! I pushed an update that references tags only.